src/field_theory/ratfunc.lean

```diff
@@ -473,6 +475,14 @@ protected lemma induction_on {P : ratfunc K → Prop} (x : ratfunc K)
   P x :=
 x.induction_on' (λ p q hq, by simpa using f p q hq)
 
+/-- `ratfunc K` is isomorphic to the field of fractions of `polynomial K`, as algebras.
+
+This is an auxiliary definition; `simp`-normal form is `is_localization.alg_equiv`.
+-/
+@[simps] def to_fraction_ring_alg_equiv : ratfunc K ≃ₐ[K] fraction_ring (polynomial K) :=
```

Comment:
I'm not sure what the point is of this definition compared to `(fraction_ring.alg_equiv (polynomial K) (ratfunc K)).restrict_scalars K`. Especially since the docstring suggests to use that one instead.

Reply:
That makes sense, I will delete the new def. Is there a way to make that `alg_equiv` more discoverable? I would have assumed there would have been something in the `ratfunc` namespace to convert back to the underlying definition. The `alg_equiv` you propose is in the `fraction_ring` namespace and requires a symm:
```
(fraction_ring.alg_equiv (polynomial K) (ratfunc K)).symm.restrict_scalars K
```
Should I add this example to `Implementation Notes`?